lib/ios/TopBarPresenter.m

```diff
@@ -112,18 +112,21 @@ - (void)setLargeTitleAttributes:(RNNLargeTitleOptions *)largeTitleOptions {
 }
 
 - (void)setBackButtonOptions:(UIImage *)icon withColor:(UIColor *)color title:(NSString *)title showTitle:(BOOL)showTitle fontFamily:(NSString *)fontFamily fontSize:(NSNumber *)fontSize {
-	UIBarButtonItem *backItem = [[UIBarButtonItem alloc] init];
-    NSArray* stackChildren = self.navigationController.viewControllers;
+	NSArray* stackChildren = self.navigationController.viewControllers;
+    UIViewController *lastViewControllerInStack = stackChildren.count > 1 ? stackChildren[stackChildren.count - 2] : self.navigationController.topViewController;
```

Comment:
If the stack has a single child, we set the back button text from the single child it contains. But it doesn't have a back button in this case - so why run this logic if `count <= 1` ?